apps/src/templates/teacherDashboard/SectionsPage.jsx

```diff
@@ -43,6 +43,10 @@ class SectionsPage extends Component {
     rosterDialogOpen: false,
   };
 
+  componentWillMount() {
+    this.provider = experiments.isEnabled('googleClassroom') ? 'google' : 'clever';
```

Comment:
Looks like elsewhere you check `cleverClassroom` experiment. Presumably that's needed here? Will we not ever be in a situation where we want both options?